vllm/utils.py

```diff
@@ -425,6 +426,10 @@ def is_hip() -> bool:
     return torch.version.hip is not None
 
 
+def is_navi4x() -> bool:
```

Comment:
I suggest using an lru_cache here, and accessing envs through envs.py
Also, relying on this env in runtile may be problematic, it's subject to change, regardless of what vllm was compiled like
It's better to make this a platform check, see platforms/rocm.py